### PR TITLE
There What you asked for

### PR DESCRIPTION
**Pull request by taelon875**
Only the alser and useperkfixes

### CODE REVIEW

File: SWLOR.Game.Server/Feature/UsePerkFeat.cs [hunk 1]
```diff
@@ -273,15 +273,6 @@ void CompleteActivation(string id, float abilityRecastDelay)
             CheckForActivationInterruption(activationId, position);
             SetLocalInt(activator, activationId, (int)ActivationStatus.Started);
 
-            if (GetIsPC(activator) && !Space.IsPlayerInSpaceMode(activator))
-            {
-                ApplyEffectToObject(DurationType.Temporary, EffectSlow(), activator, 1.5f);
-                DelayCommand(1.6f, () =>
-                {
-                    Stat.ApplyPlayerMovementRate(activator);
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/UsePerkFeat.cs:281]: Are you sure this isn't needed?

Comment (taelon875) [reply SWLOR.Game.Server/Feature/UsePerkFeat.cs:281]: 95% sure?  The only things that are in space that would trigger this are things with activation delays... And nothing but mining lasers have those. And honestly you shouldnt be moving while mining. So i felt it solved two issues there.

Comment (zunath) [reply SWLOR.Game.Server/Feature/UsePerkFeat.cs:281]: Well, no. You're still immobilizing the player on line 287 below which affects movement speed. The reapplication of the movement rate is there to make sure that immobilization (or any kind of slow) doesn't permanently reduce speed. Can you double check this by turning on Dash, using an ability, then making sure your movement rate is still the same as it was before the ability usage?

Comment (taelon875) [reply SWLOR.Game.Server/Feature/UsePerkFeat.cs:281]: I mean 287 is the one i added after this was removed, and in my testing I didn't notice any reduction in speed. I will apply a movement speed back after.

**Comment by taelon875** [comment]
Okay... After a few hours I finally got the good version working... Thanks for the help Zun, I got there in the end though!

**Review by zunath — APPROVED** [review]

**Pull request merged**